pages/common/uv-add.md

```diff
@@ -0,0 +1,37 @@
+# uv add
+
+> Add package dependencies to the project.
```

Comment:
Please specify what type of project this is for.

Reply:
Resolved in the latest commit